README.md

```diff
@@ -475,6 +475,9 @@ Other Style Guides
     // bad
     const name = "Capt. Janeway";
 
+    // bad
```

Comment:
And I guess there's also tagged template literals, which might contain neither interpolation nor newlines.